proto/desmos/profiles/v1beta1/models.proto

```diff
@@ -92,15 +94,48 @@ message UserBlock {
   option (gogoproto.equal) = true;
   option (gogoproto.goproto_stringer) = true;
 
-  string blocker = 1 [(gogoproto.moretags) = "yaml:\"blocker\""];
-  string blocked = 2 [(gogoproto.moretags) = "yaml:\"blocked\""];
+  string blocker = 1 [ (gogoproto.moretags) = "yaml:\"blocker\"" ];
+  string blocked = 2 [ (gogoproto.moretags) = "yaml:\"blocked\"" ];
   string reason = 3 [
     (gogoproto.moretags) = "yaml:\"reason\""
   ]; // Reason the user has been blocked for (optional).
-  string subspace = 4 [(gogoproto.moretags) = "yaml:\"subspace\""];
+  string subspace = 4 [ (gogoproto.moretags) = "yaml:\"subspace\"" ];
 }
 
 // UserBlocks wraps a list of UserBlock objects
 message UserBlocks {
-  repeated UserBlock blocks = 1 [(gogoproto.nullable) = false];
-}
\ No newline at end of file
+  repeated UserBlock blocks = 1 [ (gogoproto.nullable) = false ];
+}
+
+// ___________________________________________________________________________________________________________________
+
+// Proof contains all the data used to verify the signature when linking an
+// account to a profile
+message Proof {
+  google.protobuf.Any pub_key = 1 [ (gogoproto.moretags) = "yaml:\"pub_key\"" ];
+  string signature = 2 [ (gogoproto.moretags) = "yaml:\"signature\"" ];
+}
+
+// ChainConfig contains the data of the chain with which the link is made.
+message ChainConfig {
+  string id = 1
+      [ (gogoproto.moretags) = "yaml:\"id\"", (gogoproto.customname) = "ID" ];
```

Comment:
Yesterday I was thinking more about this ID and whether we should keep it as such or changing it to something more like `name`. I thought of this cause it might happen that some chains (.ie. Solana, Polkadot, etc) might have different ids for different versions while the name (i.e. `solana`, `polkadot`) stays the same. 

What do you guys think @dadamu @bragaz? Should we keep it referencing the chain id (e.g `desmos-mainnet-1`) or should we switch to a more generic chain name (e.g. `desmos`)?

Reply:
I think generic chain name is better because a key would generate same address string in the same chain system no matter it is test-net or main-net.

Reply:
Yes I think a `name` field is even better understandable from the devs that will use it later during implementations.